e2e-cypress/files/README.md

```diff
@@ -10,21 +10,28 @@ Please note that each stage is executed with its own cypress configuration file
 
 ## Running end-to-end tests
 
-Run `npm run e2e` to execute all end-to-end tests via [cypress](http://www.cypress.io) against the test instance of the front end.
+Run `npm run e2e` to execute all end-to-end tests via [cypress](http://www.cypress.io) against the test instance of the front end. (This includes a cleanup task called `delete-junit-results` which only runs in a UNIX-based shell. On a Windows command line you will probably get an error around `rm` not being found, please adapt `delete-unit-results` accordingly in that case.)
```

Comment:
I once saw a little helper called ``rimraf`` (https://github.com/isaacs/rimraf). It is also available as npm package. When I remeber it correctly someone can use it **OS-independently**. After adding it to the dev dependencies (``npm install rimraf --save-dev``) it should be possible to use it like this:
```
  ...
  "delete-junit-results": "rimraf build/test-results``,
  ...
```

Reply:
Thanks for the suggestion! I tested it on my Mac and on Jenkins, works fine on both so I included it.